media/js/base/mozilla-image-helper.js

```diff
@@ -38,8 +34,7 @@ Mozilla.ImageHelper.initPlatformImages = function() {
     $('.platform-img').each(function() {
         var $img = $(this);
         var data_attribute = 'src-';
-        var is_high_res = $img.data('high-res') && Mozilla.ImageHelper.isHighDpi();
-        var suffix;
+        var is_high_res = $img.data('high-res');
```

Comment:
We still need it when processing high res platform images. Platform images begin with an empty `src` attribute and no `srcset` attribute. We use JS to set `src` based on the current user's platform. If the platform image is marked with `data-high-res`, then the JS also adds/sets the `srcset` attribute.